### PR TITLE
samples: add ARMArchitecture

### PR DESCRIPTION
**Pull request by yselkowitz**
/cc @gabemontero

### CODE REVIEW

File: samples/v1/types_config.go [hunk 1]
```diff
@@ -138,7 +138,10 @@ const (
 	// of this operator, it is equivalent to X86Architecture, which is kept for historical/migration
 	// purposes
 	AMDArchitecture = "amd64"
-	// PPCArchitecture is the value used to specify the x86_64 hardware architecture
+	// ARMArchitecture is the value used to specify the aarch64 hardware architecture
+	// in the Architectures array field.
+	ARMArchitecture = "arm64"
```
Comment (adambkaplan) [inline samples/v1/types_config.go:143]: @gabemontero correct me if I'm wrong, but I believe we want the values to align with golang arch values whenever possible.

Comment (yselkowitz) [reply samples/v1/types_config.go:143]: arm64 is the golang name for aarch64.

Comment (gabemontero) [reply samples/v1/types_config.go:143]: @adambkaplan we do want it to align with `runtime.GOARCH` where `runtime` is the golang package we import 

Based on @yselkowitz 's comment I'm inferring he has confirmed this.

**Pull request merged**